suzieq/poller/controller/utils/inventory_models.py

```diff
@@ -36,11 +36,13 @@ class DeviceModel(BaseModel):
 
     @validator('jump_host')
     def jump_host_validator(cls, value):
-        jump_host = value[2:] if value.startswith('//') else value
-        pattern = '([a-zA-Z0-9-.]{1,253})@([a-zA-Z0-9-.]{1,253})'
-        assert re.fullmatch(pattern, jump_host), \
-               'format username@jumphost required'
-        return value
+        '''Validate jump host format'''
+        try:
+            uri = parse_url(value)
+            assert uri.scheme is None, \
+                'format username@jumphost[:port] required'
+        except ValueError:
+            assert False, f'Invalid jumphost format: {value}'
```

Comment:
you should return value otherwise the jump_host won't be set

Reply:
Fixed